site/en/docs/extensions/mv3/sandboxingEval/index.md

```diff
@@ -64,42 +64,94 @@ named "sandbox.html". The manifest entry looks like this:
 ### Load the sandboxed file {: #load_file }
 
 In order to do something interesting with the sandboxed file, we need to load it in a context where
-it can be addressed by the extension's code. Here, [sandbox.html][8] has been loaded into the
-extension's [Event Page][9] ([eventpage.html][10]) via an `iframe`. [eventpage.js][11] contains code
-that sends a message into the sandbox whenever the browser action is clicked by finding the `iframe`
+it can be addressed by the extension's code. Here, sandbox.html has been loaded into
+an extension page via an `iframe`. The page's javascript file contains code that sends a message 
+into the sandbox whenever the browser action is clicked by finding the `iframe`
 on the page, and executing the `postMessage` method on its `contentWindow`. The message is an object
-containing two properties: `context` and `command`. We'll dive into both in a moment.
+containing two properties: `templateName`, and `command`. We'll dive into both in a moment.
```

Comment:
It looks like it has the `context` property too?

Reply:
Could we change "We'll dive into both in a moment." to something like "We'll dive in to `context` and `command` in a moment"?

Reply:
Good catch, thank you.